src/yardstick/tool/grype.py

```diff
@@ -160,18 +171,29 @@ def _install_from_path(
         cls,
         path: Optional[str],
         src_path: str,
+        use_cache: Optional[bool] = True,
+        add_version_to_path: bool = False,
         **kwargs,
     ) -> "Grype":
-        # get the description and head ref from the repo
-        src_repo_path = os.path.abspath(os.path.expanduser(src_path))
-        build_version = utils.local_build_version_suffix(src_repo_path)
-        logging.debug(f"installing grype from path={src_repo_path!r}")
-        logging.debug(f"installing grype to path={path!r}")
+        if not use_cache and path:
+            shutil.rmtree(path, ignore_errors=True)
+
         if not path:
             path = tempfile.mkdtemp()
             atexit.register(shutil.rmtree, path)
+
+        # get the description and head ref from the repo
+        src_repo_path = os.path.abspath(os.path.expanduser(src_path))
+        build_version = utils.local_build_version_suffix(src_repo_path)
+
+        if add_version_to_path:
+            path = os.path.join(path, build_version)
```

Comment:
Doesn't this already happen on line 194?

Reply:
removed